hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerSchemaEvolutionExtensive.java

```diff
@@ -200,8 +237,17 @@ protected static Stream<Arguments> testArgs() {
         for (Boolean multiLogFiles : new Boolean[]{false, true}) {
           for (Boolean shouldCluster : new Boolean[]{false, true}) {
             for (String tableType : new String[]{"COPY_ON_WRITE", "MERGE_ON_READ"}) {
-              if (!multiLogFiles || tableType.equals("MERGE_ON_READ")) {
-                b.add(Arguments.of(tableType, shouldCluster, false, rowWriterEnable, addFilegroups, multiLogFiles));
+              for (Boolean isSparkRecord : new Boolean[]{false, true}) {
+                if (!multiLogFiles || tableType.equals("MERGE_ON_READ")) {
```

Comment:
This change adds another 32 combinations.  How much testing time does it increase in CI?

Reply:
Extensive tests don't even run in CI

Reply:
Then maybe should reduce the combinations, and enable it in CI. But in a separate PR.